pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -511,6 +511,11 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
             self._multilingual_text(item, 'Title')
             self._multilingual_text(item, 'Content')
 
+        extract_dict['DisclaimerGrundbuch'] = extract_dict['ExclusionOfLiability'][0]
```

Comment:
If you do not mind, I would prefer having all in english. 😃  
I suggest DisclaimerLandRegister